File: derby/rhino/create/derby_create.js
#! /usr/bin/java org.mozilla.javascript.tools.shell.Main
// ------------------------------------------------------------------
//
//	derby_create.js
//
//						Oct/01/2012
//
// ------------------------------------------------------------------
importPackage(java.lang);
importPackage(java.sql);
load ("/var/www/data_base/common/rhino_common/sql_manipulate.js");
load ("/var/www/data_base/common/rhino_common/text_manipulate.js");

print	("*** 開始 ***");

var id_in=arguments[0];
var population_in=arguments[1];

var url = "jdbc:derby://localhost:1527/city_aaa;create=true";

var conn = DriverManager.getConnection (url);

drop_table_proc (conn);
create_table_proc (conn);
insert_data_proc (conn);

conn.close();

print	("*** 終了 ***");
// ------------------------------------------------------------------
function insert_data_proc (conn)
{
	insert_record_proc (conn,"t3551","山口",41567,"1951-2-15");
	insert_record_proc (conn,"t3552","下関",34723,"1951-6-24");
	insert_record_proc (conn,"t3553","萩", 65428,"1951-5-27");
	insert_record_proc (conn,"t3554","徳山",27254,"1951-9-29");
	insert_record_proc (conn,"t3555","岩国",19637,"1951-3-16");
	insert_record_proc (conn,"t3556","光",75236,"1951-11-24");
	insert_record_proc (conn,"t3557","長門",49218,"1951-4-3");
	insert_record_proc (conn,"t3558","防府",46281,"1951-12-18");
	insert_record_proc (conn,"t3559","宇部",72546,"1951-3-9");
}

// ------------------------------------------------------------------

File: derby/jython/create/derby_create.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	mysql_create.py
#
#					Oct/01/2012
#
# --------------------------------------------------------
import	sys
import	string
from java.lang import	System
from java.lang import	String
#
import java
from java.sql import DriverManager
#
sys.path.append ('/var/www/data_base/common/jython_common')
sys.path.append ('/var/www/data_base/common/python_common')
#
from jython_rdb_manipulate import display_proc,drop_table_proc, \
		create_table_proc,table_insert_proc
#
from text_manipulate import dict_append_proc
# --------------------------------------------------------
def	data_prepare_proc ():
	dict_aa = {} 
#
	dict_aa = dict_append_proc (dict_aa,'t3551','山口'.decode ('utf-8'),75826,'2004-4-21')
	dict_aa = dict_append_proc (dict_aa,'t3552','下関'.decode ('utf-8'),17365,'2004-5-12')
	dict_aa = dict_append_proc (dict_aa,'t3553','萩'.decode ('utf-8'),72614,'2004-6-17')
	dict_aa = dict_append_proc (dict_aa,'t3554','徳山'.decode ('utf-8'),64328,'2004-9-9')
	dict_aa = dict_append_proc (dict_aa,'t3555','岩国'.decode ('utf-8'),42651,'2004-8-4')
	dict_aa = dict_append_proc (dict_aa,'t3556','光'.decode ('utf-8'),35187,'2004-1-21')
	dict_aa = dict_append_proc (dict_aa,'t3557','宇部'.decode ('utf-8'),81256,'2004-7-23')
	dict_aa = dict_append_proc (dict_aa,'t3558','長門'.decode ('utf-8'),24784,'2004-10-26')
	dict_aa = dict_append_proc (dict_aa,'t3559','防府'.decode ('utf-8'),75823,'2004-12-15')
#
	return	dict_aa
#
# --------------------------------------------------------
print ("*** 開始 ***")
System.out.println (String ("*** 開始 ***","utf-8"))
#
dict_aa = data_prepare_proc ()
#
url="jdbc:derby://localhost:1527/city_aaa;create=true"

conn = DriverManager.getConnection (url)
#
drop_table_proc (conn)
create_table_proc (conn)
#
table_insert_proc (conn,dict_aa)
#
#conn.commit ()
#
display_proc (conn)
#
conn.close ()
print ("*** 終了 ***")
System.out.println (String ("*** 終了 ***","utf-8"))
#
# --------------------------------------------------------
